sympy/physics/control/control_plots.py

```diff
@@ -736,18 +737,25 @@ def bode_magnitude_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs)
 
 
 def bode_magnitude_plot(system, initial_exp=-5, final_exp=5,
-    color='b', show_axes=False, grid=True, show=True, **kwargs):
+    color='b', show_axes=False, grid=True, show=True, Freq_unit='rad/sec', **kwargs):
     r"""
     Returns the Bode magnitude plot of a continuous-time system.
 
     See ``bode_plot`` for all the parameters.
     """
+    Freq_units = ('rad/sec', 'Hz')
+    if Freq_unit not in Freq_units:
+        raise ValueError('Only rad/sec and Hz are accepted frequency units.')
+
     x, y = bode_magnitude_numerical_data(system, initial_exp=initial_exp,
         final_exp=final_exp)
+    if Freq_unit == 'Hz':
```

Comment:
Ideally, this should be passed to `bode_magnitude_numerical_data`. In that way you should be able to divide by 2*pi in the SymPy expression rather than in the vector (although I have not checked how it looks there).

Reply:
Should be straightforward to obtain it here:
https://github.com/sympy/sympy/blob/69662b4fb4a9c585742da50ff33c5ba896108f23/sympy/physics/control/control_plots.py#L730

Reply:
Sorry I misjudged the hint thinking I could handle the constant in the SymPy expression itself .I'll make this change .

Reply:
@akshanshbhatt @oscargus 
I have made the changes .These are the improvements -

```
>>> from sympy.abc import s
>>> from sympy.physics.control.lti import TransferFunction
>>> tf = TransferFunction(1, 1+s, s)
>>> from sympy.physics.control.control_plots import bode_magnitude_plot
>>> bode_magnitude_plot(tf)
```
![Ip1](https://user-images.githubusercontent.com/90216905/154203713-baf75edc-676b-4479-b304-624bb1c6d132.png)

```
>>> from sympy.physics.control.control_plots import bode_plot
>>> bode_plot(tf)
```

![Ip2](https://user-images.githubusercontent.com/90216905/154203859-838ba326-3cc5-48aa-be21-cff17a22ef89.png)

`>>> bode_plot(tf,freq_unit='Hz')`

![Ip3](https://user-images.githubusercontent.com/90216905/154203969-8ef8f37a-d2dd-4f1f-bd28-de81e2c6dfab.png)

`>>> bode_plot(tf,phase_unit='deg')`

![Ip4](https://user-images.githubusercontent.com/90216905/154204038-fa3e77aa-97f1-4407-9cca-b54339ab0f81.png)